Sources/Controllers/MessagesViewController.swift

```diff
@@ -478,7 +478,8 @@ UICollectionViewDelegateFlowLayout, UICollectionViewDataSource, UIGestureRecogni
     }
     
     @objc private func clearMemoryCache() {
-        MessageStyle.bubbleImageCache.removeAllObjects()
+        // Since quipper/MessageKit does not use image caches, the next line is commented out.
+        // MessageStyle.bubbleImageCache.removeAllObjects()
```

Comment:
As I mentioned in https://github.com/quipper/MessageKit/pull/22#discussion_r1304194334, `MessageStyle` enum is completely diverged from the original.
`MessageStyle.bubbleImageCache` does not exist in our version.